modules/reference/appvms/flatpak.nix

```diff
@@ -56,6 +123,9 @@ in
         services.flatpak.enable = true;
         security.rtkit.enable = lib.mkForce true;
         services.packagekit.enable = true;
+        ghaf.xdgitems.enable = true;
+        ghaf.xdghandlers.url = true;
+        ghaf.xdghandlers.urlScript = "${urlScript}/bin/xdgflatpakurl";
```

Comment:
The original idea was that XDG items are created in VMs that need to open certain file types like PDFs, images, URLs (client side) and XDG handlers are created in VMs that know how to handle those types (server side). If the task is to open a flatpak browser instead of the default Ghaf browser in apps that run in flatpak-vm, it might be cleaner to create an XDG item for it there and set it as the default for URLs. In that item, you can use the same script to search for a browser and if it’s not found, fall back to the regular Ghaf browser.

XDG handlers need to be enabled if we want to let other VMs use the flatpak browser. In this case, the implementation of the XDG items module needs to be updated to make it more configurable and to remove the hard-coded chrome-vm and chromium-vm.